main.py

```diff
@@ -0,0 +1,260 @@
+        max_salary="80,000",
+        avg_salary="65,000",
+        education="Bachelor's or master's degree in forensic accounting, accounting, finance or a related field is required for forensic accountants. Additional education in criminal justice or law enforcement is a plus.",
+        short_description="Forensic accountants examine data to determine where missing money has gone and how to recover it. ",
+        long_description="Forensic accountants examine data to determine where missing money has gone and how to recover it. They may also present reports of their financial findings as evidence during hearings, where they often testify as expert witnesses. This work serves an important purpose at public accounting and consulting firms, law firms, law enforcement agencies, and insurance companies."
+    )
+    job14 = JobsModel(
+        title="Civil Engineer",
+        icon_url="https://www.flaticon.com/premium-icon/icons/svg/2316/2316096.svg",
+        image_url="https://image.freepik.com/free-vector/multicultural-team-civil-engineers-construction-site-flat-character-vector-illustration-concept_213110-255.jpg",
+        video_url="https://www.youtube.com/watch?v=cJaRjI7K-Lw",
+        min_salary="58,000",
+        max_salary="121,000",
+        avg_salary="89,500",
+        education="Bachelor's degree in Engineering",
+        short_description="Civil engineers design major transportation projects. ",
+        long_description="Civil engineers conceive, design, build, supervise, operate, construct and maintain infrastructure projects and systems in the public and private sector, including roads, buildings, airports, tunnels, dams, bridges, and systems for water supply and sewage treatment."
+    )
+    job15 = JobsModel(
+        title="Pharmacist",
+        icon_url="https://www.flaticon.com/svg/static/icons/svg/1256/1256535.svg",
+        image_url="https://image.freepik.com/free-vector/customers-pharmacist-pharmacy-store-people-buying-medication-drugstore-flat-vector-illustration-service-treatment-pharmaceutics-concept_74855-9835.jpg",
+        video_url="https://www.youtube.com/watch?v=0xxcQQtZKzI",
+        min_salary="107,000",
+        max_salary="151,000",
+        avg_salary="129,000",
+        education="Bachlor's degree and a Doctor of Pharmacy (PharmD).",
+        short_description="Provide medical advice and dispense medications. ",
+        long_description="Pharmacists are healthcare practitioners who check and distribute drugs to doctors for medication that had been prescribed to patients. They advise patients and health care providers on the selection, dosages, interactions, and side effects of medications. They might work in a  hospital or retail store."
+    )
+    jobs = [job1, job2, job3, job4, job5, job6, job7, job8, job9, job10, job11, job12, job13, job14, job15]
+    for job in jobs:
+        db.session.add(job)
+    db.session.commit()
+    print('Database seeded!')
+
+@app.route('/jobs')
+def handle_jobs():
```

Comment:
One of the next steps we can do this week is refactor some of these into their own files